treeherder/config/settings.py

```diff
@@ -436,3 +444,15 @@
 
 # Resource count to limit the number of threads opening connections with the DB
 CONN_RESOURCES = 50
+
+
+#  Default environment variables check
+REQUIRED_PRODUCTION_ENV_VARIABLES = ["TREEHERDER_DJANGO_SECRET_KEY", "BROKER_URL",
+                                     "DATABASE_URL", "REDIS_URL", "TREEHERDER_DEBUG",
+                                     "NEW_RELIC_DEVELOPER_MODE"]
+if not DEBUG:
```

Comment:
Please change `if not DEBUG` for `if os.environ.get("HEROKU_APP_ID")`.
As-is, `./manage.py check` [1] in a virtualenv outside of Docker thinks that I'm trying to set up a production environment.

Move this block right after "Top Level configuration".

`HEROKU_APP_NAME` is added manually when setting app Heroku, however, `HEROKU_APP_ID`.
This is a conscious decision and we can iterate in the future if we were to need a new env variable.
This meets our current needs.

[1]
```
(.venv) armenzg@Armens-MacBook-Pro treeherder % ./manage.py check
For a production set up please set the missing variables: TREEHERDER_DJANGO_SECRET_KEY, BROKER_URL, DATABASE_URL, REDIS_URL, TREEHERDER_DEBUG, NEW_RELIC_DEVELOPER_MODE
System check identified no issues (0 silenced).
```